tsl/src/compression/compression.c

```diff
@@ -1541,74 +1578,48 @@ decompress_chunk(Oid in_table, Oid out_table)
 	 */
 	Relation in_rel = relation_open(in_table, ExclusiveLock);
 
-	TupleDesc in_desc = RelationGetDescr(in_rel);
-	TupleDesc out_desc = RelationGetDescr(out_rel);
-
 	Oid compressed_data_type_oid = ts_custom_type_cache_get(CUSTOM_TYPE_COMPRESSED_DATA)->type_oid;
```

Comment:
It's the same regardless of the table, right? Makes sense to move it inside the build function as well.